### PR TITLE
add autcomplete scenes

### PR DESCRIPTION
**Pull request by annieyw**

### CODE REVIEW

**Comment by annieyw** [comment]
image

File: scenes/e2e/src/app.e2e-spec.ts [hunk 1]
```diff
@@ -25,7 +26,7 @@ describe('screenshot scenes for each component', () => {
     'slide-toggle',
     'slider',
     'stepper',
-    'snack-bar',
```
Comment (mmalerba) [inline scenes/e2e/src/app.e2e-spec.ts:28]: please leave the extra comma, makes merge conflicts easier to resolve

File: scenes/src/app/scenes/autocomplete/autocomplete-scene.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<form class="docs-autocomplete-example-form">
```
Comment (mmalerba) [inline scenes/src/app/scenes/autocomplete/autocomplete-scene.html:1]: I don't think this class is used so it can be removed

**Comment by mmalerba** [comment]
@annieyw needs rebase

**Pull request merged**